src/test/java/org/mockito/internal/session/DefaultMockitoSessionBuilderTest.java

```diff
@@ -0,0 +1,22 @@
+package org.mockito.internal.session;
+
+import org.junit.Test;
+import org.mockito.quality.Strictness;
+
+public class DefaultMockitoSessionBuilderTest {
+
+    @Test public void creates_sessions() throws Exception {
```

Comment:
It is a test for the builder, if it creates objects that we can call startMocking() and finishMocking() on, it is a sufficient verification. 

I'll improve it so that it is better and does not raise question like yours. Good feedback!